packages/modeling/src/operations/modifiers/generalize.js

```diff
@@ -67,6 +62,9 @@ const generalizeGeom3 = (options, geometry) => {
   // FIXME replace with geom3.cloneShallow() when available
   const clone = Object.assign({}, geometry)
   clone.polygons = polygons
+  // toPolygons called above will convert old polygons (apply transform and return the new polygons array)
+  // we must reset transforms to identity, otherwise the transform will be applied again in some other operation
+  clone.transforms = mat4.create()
```

Comment:
Why not use create(polygons)?

Reply:
I did not want to change the code too much, just fix it so the transform matrix matches the new situation.
The original code uses cloning, and createPolygons would only keep the polygons.
Additionally, this code is the is in the direction of cloning that I am also aiming for.

Reply:
Actually, a more cool javascript style would be…

`const clone = Object.assign({}, geometry, {polygons, transforms: mat4.create()})`

:)

Reply:
ok :)